servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -246,6 +245,13 @@ def format_scattered_task(task_name, task_metadata):
                     shard_index=shard.get('shardIndex'),
                     execution_events=_get_execution_events(
                         shard.get('executionEvents'))))
+            if min_start > _parse_datetime(shard.get('start')):
+                min_start = _parse_datetime(shard.get('start'))
+            if shard.get('executionStatus') not in ['Failed', 'Done']:
+                max_end = None
+            if max_end is not None and max_end < _parse_datetime(
```

Comment:
[ToL] I am notoriously bad at reasoning about double negatives - is there a `is_defined` construct of some kind we could employ instead?